.github/actions/verify-checkout/action.yaml

```diff
@@ -12,5 +12,7 @@ runs:
         git_sha=$(git log -1 --format='%H')
         if [[ "$git_sha" != "$GITHUB_SHA" ]]; then
             echo "mismatch git sha $git_sha != $GITHUB_SHA"
+            echo "logs:"
+            git log
```

Comment:
Is this going to output all the commits? maybe something like `git log -20` to limit the length of the output?

Reply:
ouch, let me send a PR to do that.